wger/exercises/api/views.py

```diff
@@ -221,11 +221,22 @@ class ExerciseCommentViewSet(viewsets.ReadOnlyModelViewSet):
     """
     API endpoint for exercise comment objects
     """
-    queryset = ExerciseComment.objects.all()
     serializer_class = ExerciseCommentSerializer
+
     ordering_fields = '__all__'
-    filterset_fields = ('comment',
-                        'exercise')
+    filterset_fields = ('exercise',
+                        'comment'
+                        )
+    """
+    Filter by language for exercise comments
+    """
+    def get_queryset(self):
+        qs = ExerciseComment.objects.all()
+        language = self.request.query_params.get('language', None)
```

Comment:
You don't need to explicitly add None, .get() returns None by default

Reply:
Corrected this.